m/mika.md

```diff
@@ -0,0 +1,11 @@
+# Mika sy Davis
+
+![Image of mika & davis](mika.jpg)
```

Comment:
this image filename needs to match the filename of the artist.
There's already an artist called Mika, we don't want to create confusion!

Reply:
I've finished it now, so please could you check it !